app/helpers/finance_helper.rb

```diff
@@ -9,6 +23,19 @@ def float_to_percentage(number)
     number_to_percentage(number * 100, precision: 0)
   end
 
+  def aggregated_payment(breakdown_array)
+    breakdown_array.map { |breakdown|
+      breakdown[:service_fees][:monthly] + breakdown[:output_payments][:subtotal]
+    }.inject(&:+)
+  end
+
+  # TODO: The VAT flag is set on the ECF lead_provider and not on the cpd_lead_provider table. Once someone is VAT
+  # registered, then it shouldn't matter if they're one, the other or both. VAT should apply evenly. Needs to be
+  # moved and migrated from LeadProvider to CPDLeadProvider.
```

Comment:
That still needs to be addressed, I've asked in the cpd lead provider channel, probably needs to be addressed in a separate PR. Granted we want to copy what was done for ECF payment breakdowns